package.json

```diff
@@ -88,7 +88,7 @@
     "protractor": "2.2.0",
     "rx": "latest",
     "tslint": "2.5.0",
-    "typescript": "1.6.2",
+    "typescript": "1.8.0-dev.20151017",
```

Comment:
relates to https://github.com/Microsoft/TypeScript/issues/5269#issuecomment-148947611 , bump up version including discussed changes.